src/pages/index.tsx

```diff
@@ -19,6 +23,23 @@ function Page(props: Props) {
 
   const title = site?.siteMetadata?.title ?? ''
   const siteUrl = `https://${host}${pathname}`
+  const perPage = 10
+  const productList = useProductsQuery({
+    first: perPage,
+    after: (perPage * 1).toString(),
+    sort: 'score_desc',
+    term: '',
+    selectedFacets: [{ key: 'category-1', value: 'office' }],
```

Comment:
Are these selected facets would be intended to be hardcoded?

Reply:
At this moment in an alpha version, this query is to get a real example about how to get products on the homepage.
As we define the products to be shown we can change the query.